schemas/kits-schema.json

```diff
@@ -35,6 +35,22 @@
                 "type": "string",
                 "description": "Architecture to target"
             },
+            "environmentVariablesShellScript": {
+                "type": "string",
+                "description": "Filename of a shell script which sets environment variables for the kit"
+            },
+            "environmentVariablesShellScriptDirectory": {
+                "type": "string",
+                "description": "Directory in which to change before running the shell script"
+            },
+            "environmentVariablesShellScriptArgs": {
```

Comment:
There are a lot of rules around quotes and escaping of arguments for the different shells which don't appear to be handled in this PR.  Do we really want to try to emulate those behaviors, or can we get by with just supporting a script path?

Reply:
A script path should be fine.